server/oneoff/add_buffs.py

```diff
@@ -0,0 +1,66 @@
+import sys
+
+sys.path.append("../")
```

Comment:
I think you don't need this if you run it using `python -m oneoff.add_buffs` while in `server/`